src/azure-cli/azure/cli/command_modules/storage/_client_factory.py

```diff
@@ -143,7 +143,8 @@ def cf_sa_for_keys(cli_ctx, _):
     logger = get_logger(__name__)
     logger.debug('Disable HTTP logging to avoid having storage keys in debug logs')
     client = storage_client_factory(cli_ctx)
-    client.config.enable_http_logger = False
+    if hasattr(client, 'config'):
+        client.config.enable_http_logger = False
```

Comment:
In track2, you want to pass `logging_enable=False` as kwargs of the operation instead (kwargs of list_keys if I understand this usage correctly).

Reply:
Got it.

Reply:
Since you're not in core, and this code is not generic (it's storage specific), you shouldn't need any "is_track2" test, you know what version of SDK you are depending on.
So it's not incorrect, but unnecessary technical debt to me